server/athenian/api/controllers/features/github/pull_request_filter.py

```diff
@@ -554,17 +553,25 @@ async def fetch_pull_requests(prs: Dict[str, Set[int]],
     for k, v in unreleased.items():
         if v is not None and k not in facts:
             facts[k] = v
+
+    filtered_prs = []
     with sentry_sdk.start_span(op="PullRequestFactsMiner.__call__",
                                description=str(len(prs))):
         facts_miner = PullRequestFactsMiner(await bots(mdb))
         pdb_misses = 0
         for pr in prs:
             node_id = pr.pr[PullRequest.node_id.key]
             if node_id not in facts:
-                facts[node_id] = facts_miner(pr)
-                pdb_misses += 1
+                try:
+                    facts[node_id] = facts_miner(pr)
+                except ImpossiblePullRequest:
+                    print("ImpossiblePullRequest")
```

Comment:
Right, this endpoint is not used in the current production, so if `ImpossiblePullRequest` is properly handled in e.g. `/filter/pull_requests`, it was ignored here.

```suggestion
                    continue
```